snowflake/cmd/policyRecommendation.go

```diff
@@ -204,8 +204,8 @@ var policyRecommendationCmd = &cobra.Command{
 	Use:   "policy-recommendation",
 	Short: "Run the policy recommendation UDF in Snowflake",
 	Long: `This command runs the policy recommendation UDF in Snowflake. 
-You need to bring your own Snowflake account and created the policy
-recommendation UDF using the create-udfs command first.
+You need to bring your own Snowflake account and must have the policy
+recommendation UDF created using the create-udfs command first.
```

Comment:
Just realized UDF creation has been included in the onboard command. Could you help change it to `must have run the onboard command first`

Reply:
Check. I am not sure if "must have done sth" can also mean what we want here. Let me know if you are sure about that.

Reply:
Current version looks good to me, `must have` and `need to` are a bit repetitive.